app/charts/index.ts

```diff
@@ -278,9 +278,10 @@ export const getInitialConfig = ({
             componentIri: geoShapes[0]?.iri || "",
             measureIri: measures[0].iri,
             hierarchyLevel: 1,
+            colorScaleType: "continuous",
+            colorScaleInterpolationType: "linear",
             palette: "oranges",
             nbClass: 5,
-            paletteType: "continuous",
```

Comment:
I guess that if we open a chart with an old config with this new change, we will not be able to open the new chart right ? I wonder how to manage breaking changes within the config.
The best would be to migrate the config before ingestion into io-ts I guess.